specs/AgriFood/AgriFarm/example-normalized-ld.jsonld

```diff
@@ -0,0 +1,49 @@
+{
+    "@context": [
+        "http://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld",
+        "https://schema.lab.fiware.org/ld/jsonldcontext.jsonld"
+    ],
+    "id": "urn:ngsi-ld:AgriFarm:72d9fb43-53f8-4ec8-a33c-fa931360259a",
+    "type": "AgriFarm",
+    "createdAt": "2017-01-01T01:20:00Z",
+    "modifiedAt": "2017-05-04T12:30:00Z",
+    "name": {
+        "type": "Property",
+        "value": "Wheat farm"
+    },
+    "description": {
+        "type": "Property",
+        "value": "A farm producing wheat"
+    },
+    "location": {
```

Comment:
are you sure we need location for AgriFarm? AF I remember from our last meeting not, as the AgriFarm is just a collection of buildings, parcels, anyways we can discuss during our meeting tomorrow

Reply:
Not completely sure, but I have seen interesting having the possibility of providing the polygon of the whole farm in those scenarios where the granularity of parcels and buildings is not needed. (Maybe it should be an optional field)

At the end of the day even is a farm entity is a collection of parcels and buildings you are conceptually representing a real farm.